decoder.rb

```diff
@@ -0,0 +1,18 @@
+message = '.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...'
+ALPHABET = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G', '....' => 'H',
+  '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+  '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+  '-.--' => 'Y', '--..' => 'Z'
+}.freeze
+
+def decode(message)
+  output_message = ''
+  message.split('   ', -1).map { |item| item.split(' ', -1) }.each do |word|
+    word.each { |letter| output_message += ALPHABET[letter] }
+    output_message += ' '
+  end
+  puts output_message
+end
+
+decode message
```

Comment:
- [OPTIONAL] Smart solution there. Please it will be nice to also implement the solution using three methods as suggested in the project requirements so as  to practice reusing methods. 

![reusingMethods](https://user-images.githubusercontent.com/65852621/155417590-8f74dec6-d2aa-456b-aea1-29653b6e2999.png)